src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1288,6 +1309,7 @@ static void sub_6E1F34(
                         z += gSceneryShiftPressZOffset;
 
                         z = std::max<int16_t>(z, 16);
+                        z = std::min<int16_t>(z, maxPossibleHeight);
```

Comment:
`std::min` and `std::max` can be combined with `Math::Clamp`

Reply:
We should also be able to use `std::clamp` now that we're on C++17. `Math::Clamp` should probably be phased out entirely. Note that the order of arguments differs.

Reply:
Right, that'd be even nicer! Technically the order doesn't really matter, as long as the last argument is the max, but it'd be nicer to match the values with the argument's names of course.